.github/actions/setup-environment/action.yaml

```
@@ -0,0 +1,21 @@
+name: "Setup Environment"
```

yamllint

[error] wrong new line character: expected \n

(new-lines)